### ISSUE TITLE
Accessibility Issue: Missing Aria-labelledby(s)

### ISSUE THREAD

**Issue by wakingmorning** [badcamp/badcamp-2018#166]
The main navigation has an aria-labelledby attribute that points to an element ID that does not seem to exist. 

`aria-labelledby="block-mainnavigation-menu"`
<img width="497" alt="screen shot 2018-08-21 at 4 06 47 pm" src="https://user-images.githubusercontent.com/6785398/44426353-6aab8500-a55c-11e8-9d5b-7096fb66ea60.png">

There is no element with the ID `block-mainnavigation-menu`

The Pagination is also referencing a non-existent aria-labelledby.

<img width="465" alt="screen shot 2018-08-21 at 4 11 08 pm" src="https://user-images.githubusercontent.com/6785398/44426547-e60d3680-a55c-11e8-8b70-176ae29f31f3.png">

URL: https://2018.badcamp.net/session-topics/architecture

There is no element with the ID `pagination-heading` on the page
